source/Octo.Tests/Octo.Tests.csproj

```diff
@@ -29,7 +29,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Nancy" Version="2.0.0-clinteastwood" />
-    <PackageReference Include="Octopus.Client" Version="8.8.1" />
+    <PackageReference Include="Octopus.Client" Version="8.8.2-bug-promote-release4" />
```

Comment:
Remember to use the full release client before merging.

Reply:
done